webcompat/templates/shared/footer.html

```diff
@@ -1,40 +1,94 @@
-<footer class="wc-Footer" role="contentinfo">
-  <div class="wc-UIContent wc-UIContent--footer">
-    <div class="wc-Footer-wrapper">
-      <ul class="wc-Footer-list r-ResetList">
-        <li class="wc-Footer-item">
-          <a class="wc-Footer-link wc-Footer-link--separator" href="{{ url_for('about') }}">
-            <span class="wc-Footer-link-label">About</span>
+<footer id="footer" class="grid">
+  <section class="grid-row">
+      <ul class="grid-cell x2 footer-list">
+        <li class="footer-item{% if request.url_rule.endpoint == 'index' %} active{% endif %}">
+          <a class="footer-item-link" aria-label="go to home" href="{{ url_for('index') }}">
+            Home
           </a>
         </li>
-        <li class="wc-Footer-item">
-          <a class="wc-Footer-link wc-Footer-link--separator" href="{{ url_for('privacy') }}">
-            <span class="wc-Footer-link-label">Privacy Policy</span>
+        <li class="footer-item{% if request.url_rule.endpoint == 'show_issues' %} active{% endif %}">
+          <a class="footer-item-link" aria-label="go to list of isses" href="{{ url_for('show_issues') }}">
+            List of issues
           </a>
         </li>
-        <li class="wc-Footer-item">
-          <a class="wc-Footer-link wc-Footer-link--separator" href="/humans.txt">
-            <span class="wc-Footer-link-label">Contact</span>
+        <li class="footer-item{% if request.url_rule.endpoint == 'about' %} active{% endif %}">
+          <a class="footer-item-link" aria-label="go to about" href="{{ url_for('about') }}">
+            About
           </a>
         </li>
-        <li class="wc-Footer-item">
-          <a class="wc-Footer-link" href="{{ url_for('contributors') }}">
-            <span class="wc-Footer-link-label">Contribute</span>
+        <li class="footer-item{% if request.url_rule.endpoint in
+        ['contributors_build_tools', 'contributors_web_platform_research', 'contributors_organize_webcompat_events', 'contributors', 'contributors_report_bug', 'contributors_reproduce_bug', 'contributors_diagnose_bug', 'contributors_site_outreach'] %} active{% endif %}">
```

Comment:
There's probably a more elegant way to do this, maybe just `if request.url_rule.endpoint.startswith('contributors') ....` (or changing the name of the routes so it would be possible, I see there's a few groupings down below which would make the `startswith` approach problematic)

Worth filing a follow up bug for.

Reply:
Filed https://github.com/webcompat/webcompat.com/issues/2337.